extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/Utils.java

```diff
@@ -214,4 +221,44 @@ public static void sendTelemetry(JavaLanguageClient client, Object message) {
     client.sendNotification(new ExecuteCommandParams("_java.gradle.buildServer.sendTelemetry",
         Arrays.asList(message)));
   }
+
+  /**
+   * Extracts the jar file from the given artifact.
```

Comment:
Extracts the jar file from the aar file, since JDT.LS is not able to understand the structure of aar file.

Reply:
Addressed in https://github.com/microsoft/vscode-gradle/pull/1594/commits/4f6265177d1904c5c2a49da8347b1b2d547b0327